File: Java/백준/Bronze/19944. 뉴비의 기준은 뭘까？/뉴비의 기준은 뭘까？.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int b = in.nextInt();
        if(b<=2){
            System.out.println("NEWBIE!");
        }
        else if(b<=a){
            System.out.println("OLDBIE!");
        }
        else{
            System.out.println("TLE!");
        }
    }
}

File: C#/백준/Bronze/19944. 뉴비의 기준은 뭘까？/뉴비의 기준은 뭘까？.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] a = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            if(a[1]<=2){
                Console.WriteLine("NEWBIE!");
            }
            else if(a[1]<=a[0]){
                Console.WriteLine("OLDBIE!");
            }
            else{
                Console.WriteLine("TLE!");
            }
        }
    }
}